src/extractors/custom/wired.jp/index.js

```diff
@@ -0,0 +1,41 @@
+import URL from 'url';
+
+export const WiredJpExtractor = {
+  domain: 'wired.jp',
+
+  title: {
+    selectors: ['h1.post-title'],
+  },
+
+  author: {
+    selectors: ['p[itemprop="author"]'],
+  },
+
+  date_published: {
+    selectors: ['time'],
+    format: 'YYYY.MM.DD dddd HH:mm',
+    timezone: 'Asia/Tokyo',
+  },
+
+  dek: null,
+
+  lead_image_url: {
+    selectors: [['meta[name="og:image"]', 'value']],
+  },
+
+  content: {
+    selectors: ['article.article-detail'],
+
+    transforms: {
+      'img[data-original]': $node => {
+        const dataOriginal = $node.attr('data-original');
+        const src = $node.attr('src');
+        const url = URL.parse(src);
+        const origin = `${url.protocol}//${url.hostname}`;
```

Comment:
Could you please look into using something like `URL.resolve(src, dataOriginal)` so that you wouldn't have to rebuild the entire URL?

Reply:
Thank you!
`URL.resolve (src, dataOriginal)` worked well.